backend/src/utils/interfaces.ts

```diff
@@ -0,0 +1,25 @@
+//TODO find a way to enforce regex format for email inside an interface.
+export interface Company extends Object {
+  email: string;
+  orgnum: bigint;
+  username: string;
+  salt: string;
+  passhash: string;
+  name: string;
+}
+
+export interface CompanyUser extends Object {
+  email: string;
+  telephone?: bigint;
```

Comment:
might as well require it

Reply:
I can update it to be required for both the db and the interface.